src/style.css

```diff
@@ -1,3 +1,75 @@
+html {
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  background-color: aquamarine;
+}
+
+header {
+  background-color: white;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.moyo__logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+ul {
```

Comment:
Fix it in all selectors:
![image](https://github.com/mate-academy/layout_moyo-header/assets/64101898/9ab0c141-f837-426e-b7ee-35974ee3487e)

Reply:
still not fixed
<img width="636" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/b8924c16-67e2-41b1-b659-c10a950c16e3">